lib/charms/layer/apache_zeppelin.py

```diff
@@ -168,3 +170,22 @@ def close_ports(self):
     def cleanup(self):
         self.dist_config.remove_dirs()
         unitdata.kv().set('zeppelin.installed', False)
+
+
+class ZeppelinAPI(object):
+    """
+    Helper for interacting with the Appache Zeppelin REST API.
+    """
+    def _url(self, *parts):
+        url = 'http://localhost:9090/api/'
```

Comment:
Agree. Let us not hard code any of the things.

Also: do we need to deal with the case where there are multiple zeppelins, and one is acting as a master?